src/core/pyproxy.ts

```diff
@@ -361,31 +393,39 @@ class PyProxyClass {
    * default does a deep conversion, if a shallow conversion is desired, you can
    * use ``proxy.toJs({depth : 1})``. See :ref:`Explicit Conversion of PyProxy
    * <type-translations-pyproxy-to-js>` for more info.
-   *
-   * @param {object} options
-   * @param {number} [options.depth] How many layers deep to perform the
-   * conversion. Defaults to infinite.
-   * @param {array} [options.pyproxies] If provided, ``toJs`` will store all
-   * PyProxies created in this list. This allows you to easily destroy all the
-   * PyProxies by iterating the list without having to recurse over the
-   * generated structure. The most common use case is to create a new empty
-   * list, pass the list as `pyproxies`, and then later iterate over `pyproxies`
-   * to destroy all of created proxies.
-   * @param {boolean} [options.create_pyproxies] If false, ``toJs`` will throw a
-   * ``ConversionError`` rather than producing a ``PyProxy``.
-   * @param {boolean} [options.dict_converter] A function to be called on an
-   * iterable of pairs ``[key, value]``. Convert this iterable of pairs to the
-   * desired output. For instance, ``Object.fromEntries`` would convert the dict
-   * to an object, ``Array.from`` converts it to an array of entries, and ``(it) =>
-   * new Map(it)`` converts it to a ``Map`` (which is the default behavior).
-   * @return {any} The JavaScript object resulting from the conversion.
+   * @param options
+   * @return The JavaScript object resulting from the conversion.
    */
```

Comment:
Would we still need the parameter description that was here previously?

Reply:
This was one of the changes made to try to improve the rendered docs.